examples/minimal-solidstart/package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "solid-start dev --port 9898",
     "start": "solid-start start --port 9898",
     "build": "solid-start build",
-    "lint": "eslint . --ext .ts,.tsx"
+    "lint": "eslint . --ext .ts,.tsx",
+    "typecheck": "tsc --noEmit"
```

Comment:
I added this since it was missing, and it seems like we got some issues. Not sure if it's related to this PR though:

![CleanShot 2023-09-15 at 10.36.48@2x.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/J4D6C4G5z9gbCoNfTdMt/9c0bae43-0efd-40a1-a945-99423e4849e3/CleanShot%202023-09-15%20at%2010.36.48@2x.png)

Reply:
My guess is that it is because we have imports from both `react` and `solid` in single file `internal/component-theming.ts`
<img width="600" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/55754540/46c477aa-50e3-402b-9f05-2a81ed83e2f9">

And since it is being imported in solid components, which are imported in solid-start example, it results in such an issues

Reply:
I dont think so since we do that on the server for a bunch of frameworks?

Hmm would `import("react").Reactnode help?

Reply:
Hm, import is an async function. How do you intend on using it? I mean we also refer to ReactNode only as a type, nothing to do with it during the runtime

Reply:
And ESLint does not like ;D
![Uploading image.png…]()

Reply:
Made it like that but error is still there
<img width="846" alt="image" src="https://github.com/pingdotgg/uploadthing/assets/55754540/274d8cb0-d879-4163-b43f-d3c42b862155">

Reply:
I enabled "skipLibCheck" for solid start and updated "solid-start" package to latest version. Locally it fixed the issue